components/abstractions/src/main/java/com/microsoft/kiota/RequestAdapter.java

```diff
@@ -80,28 +81,28 @@ public interface RequestAdapter {
     /**
      * Executes the HTTP request specified by the given RequestInformation and returns the deserialized enum value.
      * @param requestInfo the request info to execute.
-     * @param targetClass the class of the response model to deserialize the response into.
      * @param errorMappings the error factories mapping to use in case of a failed request.
+     * @param enumParser a parser from string to enum instances.
      * @param <ModelType> the type of the response model to deserialize the response into.
      * @return the deserialized primitive response model.
      */
     @Nullable <ModelType extends Enum<ModelType>> ModelType sendEnum(
             @Nonnull final RequestInformation requestInfo,
-            @Nonnull final Class<ModelType> targetClass,
-            @Nullable final HashMap<String, ParsableFactory<? extends Parsable>> errorMappings);
+            @Nullable final HashMap<String, ParsableFactory<? extends Parsable>> errorMappings,
```

Comment:
I'm not sure about the ordering change, this is going to be inconsistent with the object methods?

Reply:
The ordering change has been done due to the linter complaining about Kotlin compatibility.
It's not consistent at the moment, should I make it consistent in this way:
- requestInfo
- errorMappings
- additionalthings
?

Reply:
sure, let's take this opportunity to align things and clean the lambda last suppressions, thanks for calling this out

Reply:
done